components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/statshandler/GatewayStatsUpdater.java

```diff
@@ -93,7 +102,9 @@ public void updateStatPublishGateway(String receiverUrl, String user, String pas
             //send the cluster message to other nodes in the cluster to update stats publishing status
             ClusteringAgent clusteringAgent = this.getConfigContext().getAxisConfiguration().getClusteringAgent();
             if (clusteringAgent != null) {
-                log.debug("Sending cluster message to Gateway domain to update stats publishing status.");
+                if(log.isDebugEnabled()) {
```

Comment:
Has been fixed in : https://github.com/wso2/carbon-apimgt/commit/92c79ab785bedab88212d711f66314b924bc041e